index.html

```diff
@@ -280,6 +280,7 @@ <h2 class="section-heading">#JoinTheDude</h2>
         <div class="container">
             <div class="row">
                 <div class="col-lg-8 col-lg-offset-2 text-center">
+                    <p></p>Learn more at <a href="https://dudesolutions.com/">DudeSolutions.com</a></p>
```

Comment:
Should this really be HTTPS?  Also, should it be a cap D and S? Or just dudesolutions.com?